apps/passport-client/src/dispatch.ts

```diff
@@ -756,7 +774,13 @@ async function doSync(
     state.pcds,
     state.subscriptions
   );
+
```

Comment:
I can't comment on the line at the top of this function, but I suggest an explicit check of `userInvalid` for an early return, so that if we've already detected invalid local state, we avoid sync entirely.

Reply:
done